android/src/main/java/com/reactnativeimagecolors/ImageColorsModule.kt

```diff
@@ -78,6 +78,18 @@ class ImageColorsModule : Module() {
     }
   }
 
+  private fun verifyFallbackColor(hex: String): String {
+    if(!hex.matches(Regex("^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$"))) {
+      throw Exception("Invalid fallback hex color. Must be in the format #ffffff or #fff")
+    }
+
+    if(hex.length == 7) {
+      return hex
+    }
+
+    return "#${hex[1]}${hex[1]}${hex[2]}${hex[2]}${hex[3]}${hex[3]}"
```

Comment:
@KrzysztofMoch  I'm new to Kotlin. Does Kotlin start their index at 1 not 0? In JavaScript this code would look like:

```
    return "#${hex[0]}${hex[0]}${hex[1]}${hex[1]}${hex[2]}${hex[2]}"
```

Reply:
Kotlin start at index 0, but we have string like `#000` so index 0 is `#`

Reply:
Oh right! Makes sense!